doc/sphinx/develop/distribution-packages/pypi-sdist-wheel.md

```diff
@@ -0,0 +1,158 @@
+`SConstruct` file.
+
+You can read more about the standard fields in the `pyproject.toml` file in the [PyPA
+documentation](https://packaging.python.org/en/latest/specifications/pyproject-toml/).
+You can read more about the `scikit-build` fields in the [`scikit-build-core`
+documentation](https://scikit-build-core.readthedocs.io/en/latest/).
+
+## Wheels
+
+As mentioned above, wheels (`.whl` files) are Python's standard binary distribution
+artifact. Similar to Conda packages, wheels include compiled libraries specific to a
+particular platform, so that end users don't need to have compilers installed to use a
+package.
+
+Building Cantera's wheels is done using `build`, similar to the sdist. However, the
+wheel is built from the unpacked sdist tarball, and you must pass the `--wheel` flag to
+`build`. Then, `build` will install `scikit-build-core` to manage building the wheel
+from Cantera's C++ and Cython source files.
+
+`scikit-build-core` uses CMake to manage the compiling and linking process for the
+wheel. Most of the configuration for the wheel build is therefore done in the
+`CMakeLists.txt` files in the `interfaces/python_sdist` folder. These are copied into
+the `build/python_sdist` folder and from there into the actual sdist tarball.
+
+```{note}
+The minimum required version of CMake is 3.24 to support all the options we use with
+`FetchContent`. A compatible version should be installed by scikit-build-core if it
+isn't available on the system.
+```
+
+There are essentially 4 stages to the wheel build:
+
+1. External dependencies (`eigen`, `fmt`, `SUNDIALS`, `HighFive`, `yaml-cpp`) are either
+   located on the system, or downloaded and compiled using the `FetchContent` mechanism
+1. Cantera's C++ source code is built and linked to the appropriate dependencies
+1. Cython runs and generates C++ source code for the Python interface
+1. The generated C++ is compiled, linked to `libcantera`, and packaged into the `.whl`
+   file
+
+```{note}
+Cantera's other dependencies, specifically Boost and libhdf5, are neither installed nor
+built during the wheel build and must be pre-installed on the build system. I couldn't
+find a way to have eigen and HighFive, respectively, find those dependencies if they
+were also installed using `FetchContent`. If Boost and libhdf5 are installed in
+non-standard directories, you can use the `Boost_ROOT` or `Boost_INCLUDE_DIRS` and
+`HDF5_ROOT` variables to specific the correct locations. See the CMake documentation for
+`find_package()`.
+```
+
+Steps 1 and 2 above are set up in the `interfaces/python_sdist/src/CMakeLists.txt` file.
+Steps 3 and 4 are set up in the `interfaces/python_sdist/cantera/CMakeLists.txt` file.
+
+## Publishing the sdist and Wheels
+
+When a version of Cantera is ready to be published to PyPI, the workflow in the
+[pypi-packages](https://github.com/Cantera/pypi-packages) repository must be executed.
+The workflow essentially conducts the steps in the [TLDR](#tldr) section above.
```

Comment:
I get a "missing cross-reference" warning for this. Do you need to add `myst_heading_anchors = N` to `conf.py`?